frontend/src/app/modules/challenges/challenges-routing.module.ts

```diff
@@ -1,10 +1,15 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { ChallengeCreationComponent } from '@modules/challenges/challenge-creation/challenge-creation.component';
 
 import { ChallengesTestPageComponent } from './challenges-test-page/challenges-test-page.component';
 import { EditorQuestionComponent } from './editor-question/editor-question.component';
 
 const routes: Routes = [
+    {
+        path: 'create',
+        component: ChallengeCreationComponent,
+    },
     {
         path: 'test',
         component: ChallengesTestPageComponent,
```

Comment:
we no longer need these routes: test, question

Reply:
we need one simple route for challenge creating: **challenges/create** instead **test** and **question**